CRM/Member/BAO/Membership.php

```diff
@@ -2125,9 +2125,12 @@ public static function setQuickConfigMembershipParameters($membershipOrg, $membe
    */
   protected static function updateDeceasedMembersStatuses() {
     $count = 0;
+
+    $deceasedStatusId = CRM_Core_PseudoConstant::getKey('CRM_Member_BAO_Membership', 'status_id', 'Deceased');
+
     // 'create' context for buildOptions returns only if enabled.
     $allStatus = self::buildOptions('status_id', 'create');
-    if (($deceasedStatusId = array_search('Deceased', $allStatus)) === FALSE) {
+    if (array_key_exists($deceasedStatusId, $allStatus) === FALSE) {
```

Comment:
I think it would be false if no key exists so if($deceasedStatusId) would suffice

Reply:
ug sorry about the duplicate comments